### DISCUSSION TITLE
What should I do after upgrade the TensorRT in my environment.

### DISCUSSION THREAD

**Discussion by binliunls** [pytorch/TensorRT#1168] [Q&A]
I upgraded the TensorRT version from 8.2.4 to 8.4.1.  Is there something that I should do on torch_tensorrt after upgrading.
Currently Environment:
PyTorch 1.11.0
CUDA 11.4
TensorRT 8.4.1.

**Comment by narendasan** [comment]
You should build off of this branch until it is merged https://github.com/pytorch/TensorRT/pull/1152

**Reply by binliunls** [reply]
Thanks that worked for me on a 3090Ti GPU. However, when I tried to compile the code by bazel using "bazel build //:libtorchtrt --compilation_mode opt" I encountered a question like below:
bazel-out/k8-opt/bin/core/lowering/passes/_objs/passes/linear_to_addmm.o:linear_to_addmm.cpp:function torch_tensorrt::core::lowering::passes::replaceLinearWithBiasNonePattern(std::shared_ptr<torch::jit::Graph>): error: undefined reference to 'torch::jit::CompilationUnit::CompilationUnit(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
Could you please help me about this issue?

**Reply by binliunls** [reply]
And below are first few lines of the error output. I think there is some directory miss.
[0 / 2] [Prepa] BazelWorkspaceStatusAction stable-status.txt
ERROR: /home/liubin/data/tool/TensorRT/cpp/bin/torchtrtc/BUILD:10:10: Linking cpp/bin/torchtrtc/torchtrtc failed: (Exit 1): gcc failed: error executing command
  (cd /root/.cache/bazel/_bazel_root/76ed39d1a73e1f2b8bf8cb35e51aa553/sandbox/processwrapper-sandbox/218/execroot/Torch-TensorRT && \
  exec env - \
    LD_LIBRARY_PATH=/opt/conda/lib/python3.8/site-packages/torch/lib:/opt/conda/lib/python3.8/site-packages/torch_tensorrt/lib:/usr/local/cuda/compat/lib:/usr/local/nvidia/lib:/usr/local/nvidia/lib64 \
    PATH=/root/.cache/bazelisk/downloads/bazelbuild/bazel-5.1.1-linux-x86_64/bin:/opt/conda/bin:/opt/conda/condabin:/usr/local/nvm/versions/node/v16.6.1/bin:/opt/conda/lib/python3.8/site-packages/torch_tensorrt/bin:/opt/conda/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin:/opt/tensorrt/bin:/opt/tools \
    PWD=/proc/self/cwd \
  /usr/bin/gcc @bazel-out/k8-opt/bin/cpp/bin/torchtrtc/torchtrtc-2.params)
# Configuration: c7a41be017bf845762bfde612cab462e1e1ea6470779bb5cb792cbb22780b6cd
# Execution platform: @local_config_platform//:host

Use --sandbox_debug to see verbose messages from the sandbox
bazel-out/k8-opt/bin/cpp/bin/torchtrtc/_objs/torchtrtc/main.o:main.cpp:function c10::IValue::toTuple() const &: error: undefined reference to 'c10::detail::torchInternalAssertFail(char const*, char const*, unsigned int, char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'

**Reply by narendasan** [reply]
Likely this is an issue with ABI. try adding `--config=pre_cxx11_abi`
